scripts/linear-ticket/tests/test_lib.py

```
@@ -0,0 +1,185 @@
+    def test_every_category_non_empty(self):
+        for category in ("no_candidate", "exists_not_linked", "policy_mismatch", "infra_error"):
+            self.assertTrue(lib.failure_guidance(category).strip())
+
+    def test_unknown_category_raises(self):
+        with self.assertRaises(KeyError):
+            lib.failure_guidance("bogus")
+
+
+class DiagnosticQuery(unittest.TestCase):
+    def test_empty_raises(self):
+        with self.assertRaises(ValueError):
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)